src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1229,23 +1229,125 @@ private int SkipBlockComment(int i)
             return i;
         }
 
-        private static char Backtick(char c)
+        private string Backtick(char c)
```

Comment:
Sorry, I wasn't clear. I meant - the only reason to have two code paths.

If the `Length > 1` code path was used unconditionally - how is that different than what you have today - that's what I meant.